engine/schema/src/com/cloud/usage/dao/UsageDaoImpl.java

```diff
@@ -469,4 +469,35 @@ public void removeOldUsageRecords(int days) {
             txn.close();
         }
     }
+
+    @SuppressWarnings("deprecation")
```

Comment:
This is mostly due to lack of documentation. I think we should not be using deprecated methods if there was a known straightforward way of moving to newer implementation. Will look into it.